tests/data/aea-config.example_multipage.yaml

```diff
@@ -0,0 +1,74 @@
+agent_name: myagent
+author: fetchai
+version: 0.2.0
+description: An example of agent configuration file for testing purposes.
+license: Apache-2.0
+aea_version: '>=0.5.0, <0.6.0'
+fingerprint: {}
+fingerprint_ignore_patterns: []
+connections:
+- fetchai/oef:0.7.0
+contracts: []
+protocols:
+- fetchai/oef_search:0.4.0
+- fetchai/default:0.4.0
+- fetchai/tac:0.5.0
+- fetchai/fipa:0.5.0
+skills:
+- fetchai/echo:0.5.0
+default_connection: fetchai/oef:0.7.0
+default_ledger: cosmos
+logging_config:
+  disable_existing_loggers: false
+  version: 1
+private_key_paths:
+   cosmos: cosmos_private_key.txt
+   ethereum: eth_private_key.txt
+connection_private_key_paths:
+   cosmos: cosmos_private_key.txt
+   ethereum: eth_private_key.txt
+registry_path: ../../packages
+---
+name: dummy
+author: dummy_author
+version: 0.1.0
+type: skill
+description: a dummy_skill for testing purposes.
+license: Apache-2.0
+aea_version: '>=0.2.0, <0.3.0'
+fingerprint:
+  __init__.py: Qmd3mY5TSBA632qYRixRVELXkmBWMZtvnxJyQC1oHDTuEm
+  behaviours.py: QmWKg1GfJpuJSoCkEKW1zUskkNo4Rsoan1AD2cXpe2E93C
+  dummy.py: QmeV6FBPAkmQC49gATSmU1aq8S1SKPv7cm2zSH8cnuqoLT
+  handlers.py: QmVe1VGT7TBdjQ5xG8X9djY77f8hebHeUeX3eDrQ6FR3yt
+  tasks.py: Qmegg4QsYSqSZN3q2zuRiBAToQ2LEiWrAPtUo7rCMrxjGJ
+fingerprint_ignore_patterns: []
+contracts: []
+protocols:
+- fetchai/default:0.1.0
+behaviours:
+  dummy:
+    args:
+      behaviour_arg_1: 1
+      behaviour_arg_2: '2'
+    class_name: DummyBehaviour
+handlers:
+  dummy:
+    args:
+      handler_arg_1: 1
+      handler_arg_2: '2'
+    class_name: DummyHandler
+  dummy_internal:
+    args:
+      handler_arg_1: 1
+      handler_arg_2: '2'
+    class_name: DummyInternalHandler
+models:
+  dummy:
+    args:
+      model_arg_1: 1
+      model_arg_2: '2'
+    class_name: DummyModel
+skills: []
+dependencies: {}
```

Comment:
I don't think we should include all the default fields here; we should only include the configurable parts...most of the above is fixed!